server/events/tests/test_models.py

```diff
@@ -0,0 +1,36 @@
+from datetime import timedelta
+from django.utils import timezone
+from django.contrib.auth import models as auth_models
+from .shared_case import SharedTestCase
+from events.models import Event
+
+class EventTest(SharedTestCase):
+    """ Test module for Event model """
+
+    def setUp(self):
+        # event1
+        Event.objects.create(
+            name='event1',
+            creator=self.user1,
+            description='this is the description for event 1',
+            location='place1',
+            start_datetime=self.start_datetime1,
+            end_datetime=self.end_datetime1)
+        # event2
+        Event.objects.create(
```

Comment:
After reviewing your testcase below, you ought to set the result of this statement to a variable, i.e. `self.event2 = Event.objects.create(...)`

Reply:
Fixed